### PR TITLE
[GITHUB-3] Prefix var names with sansible_

### PR DESCRIPTION
**Pull request by lobsterdore**
Adds sansible_ prefix to var namees.

Also updates tests, Makefile etc to latest standards and fixed
issue with Cryptopgraphy Pip module.

### CODE REVIEW

**Comment by racer1680** [comment]
"Makefile etc to latest standards"
So what is the latest standard?

**Comment by lobsterdore** [comment]
@racer1680 from the scaffold branch in sansible/sansible.

**Review by tuxpower — APPROVED** [review]
👍

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Voytek Solutions LTD
+Copyright (c) 2016
```
Comment (sreddel) [inline LICENSE:3]: Do we need to get our laywers to approve this? :)

File: Makefile [hunk 2]
```diff
@@ -1,46 +1,110 @@
...
+vagrant_%:
+	cd tests && vagrant $(subst vagrant_,,$@)
 
-integration_test_deps:
-	sed -i.bak \
-		-E 's/(.*)version: (.*)/\1version: origin\/$(BRANCH)/' \
-		tests/vagrant/integration_requirements.yml
-	rm -rf tests/vagrant/sansible.*
+## Installs a virtual environment and all python dependencies
+.venv_%:
```
Comment (sreddel) [inline Makefile:84]: This seems weird.. why accept a parameter in the target definition, but then proceed to ignore the parameter?

File: LICENSE (hunk 1 above)
Comment (lobsterdore) [reply LICENSE:3]: Haha I don't think so.

File: Makefile (hunk 2 above)
Comment (lobsterdore) [reply Makefile:84]: I think the idea here is to keep this target Ansible version agnostic.

File: tests/test.yml [hunk 3]
```diff
@@ -1,16 +1,16 @@
 ---
 
-- name: Test Ansible
+- name: Test Anaible installation
```
Comment (sreddel) [inline tests/test.yml]: Anaible?

File: Makefile (hunk 2 above)
Comment (sreddel) [reply Makefile:84]: It could just be called ".venv" in that case :)

Comment (lobsterdore) [reply Makefile:84]: Check the rest of the Makefile, that wouldn't catch calls like this:

```
make .venv_2.4.1.0
```

The idea I think (bearing in mind I didn't write the code :) ) is to enable easy testing of multiple Ansible versions.

File: tests/test.yml (hunk 3 above)
Comment (lobsterdore) [reply tests/test.yml]: Me fail English? That's unpossible!

File: Makefile (hunk 2 above)
Comment (sreddel) [reply Makefile:84]: Actually.. make checks if a file/folder of the target name exists to decide if the target needs to be processed or not.. doesnt it? In which case the wildcard in the target is appropriate.
Not a big deal. :)

Comment (lobsterdore) [reply Makefile:84]: Yep that's correct, so yeah I think this format makes sense (I blame Wojtek if not).

**Comment by sreddel** [comment]
:+1:

**Pull request merged**